Planner/src/Planner.java

```diff
@@ -322,28 +378,38 @@ public void actionPerformed(ActionEvent arg0) {
 			    	@Override
 					public void actionPerformed(ActionEvent arg0) {
 			    		int i = usersList.getIndexOf(editUser);
-			    		boolean result = UserDAO.deleteUser(editUser); 
-			    		if (result == true) {
-			    			usersList.removeElementAt(i);
-			    		}
+			    		try {
+			    			boolean result = UserDAO.deleteUser(editUser); 
+			    			if (result == true) {
+			    				usersList.removeElementAt(i);
+			    			}
+			    	
 			    		editUserFrame.dispose();
```

Comment:
I think we should add this line of code (editUserFrame.dispose()) in a "finally" block. "finally" is the block that is executed even though an exception took place or not, so it is executed either way, all the time. And this is what we want - close the editUserFrame in all cases. 
`
try {
.....
} catch (SQLException e) {

} finally {
   editUserFrame.dispose();
}
`